mala/datahandling/ldos_aligner.py

```diff
@@ -146,13 +144,24 @@ def align_ldos_to_ref(
             snapshot_ref = self.parameters.snapshot_directories_list[
                 reference_index
             ]
-            ldos_ref = np.load(
-                os.path.join(
-                    snapshot_ref.output_npy_directory,
-                    snapshot_ref.output_npy_file,
-                ),
-                mmap_mode="r",
-            )
+            if snapshot_ref.snapshot_type == 'numpy':
+                ldos_ref = np.load(
+                    os.path.join(
+                        snapshot_ref.output_npy_directory,
+                        snapshot_ref.output_npy_file,
+                    ),
+                    mmap_mode="r",
+                )
```

Comment:
Might want to use `read_from_numpy_file` here for consistency instead of raw numpy calls.